sentry_sdk/integrations/asgi.py

```diff
@@ -175,7 +176,7 @@ async def _run_app(self, scope, callback):
 
                     with hub.start_transaction(
                         transaction, custom_sampling_context={"asgi_scope": scope}
-                    ):
+                    ), start_profiling(transaction, hub):
```

Comment:
This will enable profiling for all the ASGI frameworks as well. This should be removed in #1797 as that will enable it for all transactions.